trafgen.c

```diff
@@ -216,6 +217,7 @@ static void __noreturn help(void)
 	     "  trafgen -e | trafgen -i - -o eth0 --cpp -n 1\n"
 	     "  trafgen --dev eth0 --conf fuzzing.cfg --smoke-test 10.0.0.1\n"
 	     "  trafgen --dev wlan0 --rfraw --conf beacon-test.txf -V --cpus 2\n"
+	     "  trafgen --dev wlan0 --rfraw --conf beacon-test.txf -V --cpus 2-4\n"
```

Comment:
Maybe the example could be made a bit simpler as well, to focus on the `--cpus` parameter rather than just copying the command above, e.g. `trafgen --dev eth0 --conf foo.cfg --cpus 2-4` or similar.

Also, please add the same change to the examples in `trafgen.8` as well.

Reply:
I've replaced the example withe the simpler one, it still looks like copying from the line above though. But I agree, `--cpus` option in this case gets a focus. I've added an example to `trafgen.8` also and also explicitly pointed there that the option `--cpus 2` spawns threads starting from CPU 0.